lib/req-res.js

```diff
@@ -126,7 +126,48 @@ module.exports = function ReqRes ( reqResOptions ) {
 			'data' : {}
 		} );
 
-		this.handle( callback );
+		this.handle( function ( message, respond ) {
+
+			callback( message, function ( data, msg, code ) {
+				/*
+					Support backward compatibility
+					will delete this as soon as all services will updated
+				 */
+				if ( data.status ) {
+					return respond( data );
+				}
```

Comment:
This will be a version bump, because it's not backwards compatible if you're not using the JSend spec, which is now going to be forced. Should we just remove this "backwards compatibility", and when you update to the new version you're forced to do it across the board?